Server/crafting.txt

```diff
@@ -250,7 +257,7 @@ GoldenChestplate     = GoldIngot, 1:1, 3:1, 1:2, 2:2, 3:2, 1:3, 2:3, 3:3
 IronChestplate       = IronIngot, 1:1, 3:1, 1:2, 2:2, 3:2, 1:3, 2:3, 3:3
 LeatherChestplate    = Leather,   1:1, 3:1, 1:2, 2:2, 3:2, 1:3, 2:3, 3:3
 
-# Leggins:
```

Comment:
Oh yeah, _Leggings_, my bad. It would just be nice to get item naming brought in-line with the way the game names things.